src/backend/distributed/executor/multi_executor.c

```diff
@@ -66,7 +66,8 @@ static CustomExecMethods RouterSingleModifyCustomExecMethods = {
 	.ExplainCustomScan = CitusExplainScan
 };
 
-static CustomExecMethods RouterMultiModifyCustomExecMethods = {
+/* not static to enable swapping in multi-modify logic during router execution */
+CustomExecMethods RouterMultiModifyCustomExecMethods = {
```

Comment:
I meant it can be swapped in, i.e. the compound verb "swap in". I realized later there was a garden path parse of this phrasing, but held off modifying it until someone complained. Looks like I'll change it now!